prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -8,18 +8,41 @@
 
 namespace Prime.Services
 {
+    public class EmailProps
+    {
+        public string FirstName;
+        public string LastName;
+        public string TokenUrl;
+        public int MaxViews;
+        public int ExpiryDays;
+        public string ProvisionerName;
```

Comment:
I dont know if razor requires these to be fields vs properties, but I would prefer to see these as properties

Reply:
Doesn't matter just need a model for binding so I can update these to be properties.